CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.0)
+project(kops VERSION 1.0.0)
+
+set(CMAKE_MODULE_PATH ${CMAKE_BINARY_DIR}/cmake-modules)
+
+if(NOT EXISTS "${CMAKE_BINARY_DIR}/cmake-modules/FindSDL2.cmake")
+	file(DOWNLOAD "https://github.com/tcbrindle/sdl2-cmake-scripts/raw/master/FindSDL2.cmake" "${CMAKE_BINARY_DIR}/cmake-modules/FindSDL2.cmake")
+endif()
+if(NOT EXISTS "${CMAKE_BINARY_DIR}/cmake-modules/FindSDL2_mixer.cmake")
+	file(DOWNLOAD "https://github.com/tcbrindle/sdl2-cmake-scripts/raw/master/FindSDL2_mixer.cmake" "${CMAKE_BINARY_DIR}/cmake-modules/FindSDL2_mixer.cmake")
+endif()
+if(NOT EXISTS "${CMAKE_BINARY_DIR}/cmake-modules/FindZLIB.cmake")
+	file(DOWNLOAD "https://raw.githubusercontent.com/Kitware/CMake/master/Modules/FindZLIB.cmake" "${CMAKE_BINARY_DIR}/cmake-modules/FindZLIB.cmake")
+endif()
+if(NOT EXISTS "${CMAKE_BINARY_DIR}/cmake-modules/SelectLibraryConfigurations.cmake")
+	file(DOWNLOAD "https://raw.githubusercontent.com/Kitware/CMake/master/Modules/SelectLibraryConfigurations.cmake" "${CMAKE_BINARY_DIR}/cmake-modules/SelectLibraryConfigurations.cmake")
+endif()
+if(NOT EXISTS "${CMAKE_BINARY_DIR}/cmake-modules/FindPackageHandleStandardArgs.cmake")
+	file(DOWNLOAD "https://raw.githubusercontent.com/Kitware/CMake/master/Modules/FindPackageHandleStandardArgs.cmake" "${CMAKE_BINARY_DIR}/cmake-modules/FindPackageHandleStandardArgs.cmake")
+endif()
+if(NOT EXISTS "${CMAKE_BINARY_DIR}/cmake-modules/FindPackageMessage.cmake")
+	file(DOWNLOAD "https://raw.githubusercontent.com/Kitware/CMake/master/Modules/FindPackageMessage.cmake" "${CMAKE_BINARY_DIR}/cmake-modules/FindPackageMessage.cmake")
+endif()
+
+find_package(SDL2 REQUIRED)
+find_package(SDL2_mixer REQUIRED)
+find_package(ZLIB REQUIRED)
+
+if (NOT CMAKE_BUILD_TYPE)
+	set(CMAKE_BUILD_TYPE Debug)
+endif()
+
+if (WIN32)
+	add_compile_options(/MP /MT /WX /wd4244 /wd4996)
+else()
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -funsigned-char -Wall -Wno-pedantic -Wsign-compare")
```

Comment:
Oops, these should be C-flags. But currently there are warnings preventing the build with strict warnings. I'll modify these flags before merge to at least allow build. Warnings to be fixed separately.